napari/_qt/widgets/qt_dims_slider.py

```diff
@@ -94,11 +94,17 @@ def __init__(self, parent: QWidget, axis: int):
 
     def _set_slice_from_label(self):
         """Update the dims point based on the curslice_label."""
+        # On teardown some tests fail on OSX with an `IndexError`
+        try:
+            max_allowed = self.dims.nsteps[self.axis] - 1
+        except IndexError:
+            return
+
```

Comment:
I was getting this so 🤷🏼 @sofroniewn 

 
<img width="943" alt="Screen Shot 2021-04-15 at 2 20 09 PM" src="https://user-images.githubusercontent.com/3627835/114929453-6e6d3380-9df9-11eb-95e1-59ab48abe0ef.png">